src/components/Editor/EditorMenu.js

```diff
@@ -75,14 +78,17 @@ function getMenuItems(
     event
   );
 
-  const pairedType = isOriginalId(selectedLocation.sourceId)
-    ? L10N.getStr("generated")
-    : L10N.getStr("original");
+  const isOriginal = isOriginalId(selectedLocation.sourceId);
+  const isPrettyTabOpen =
+    !isOriginal && tabs.includes(getPrettySourceURL(selectedSource.get("url")));
```

Comment:
would this fail if we were looking at a different original file?

I think we should be able to get the selectedSource, which would have the URL and ignore the tab stuff